src/common/js/timesheet/index.js

```diff
@@ -0,0 +1,244 @@
+import $ from 'jquery'
+import styles from './index.scss'
+
+// 残業申請ページ表示に要するパラメータ
+const overtimeReportState = {
+  status: 'default',
+  module: 'application_form',
+  action: 'editor',
+  application_form_master_id: '122'
+}
+// 各種申請画面
+const appFormState = {
+  module: 'application_form',
+  action: 'application_form'
+}
+
+// 残業申請画面
+const overtimeReportPath = __DEV__
+  ? 'overtime_report.html'
+  : '/'
+
+// 各種申請画面
+const appFormPath = __DEV__
+  ? 'application_form.html'
+  : `/?${$.param(appFormState)}`
```

Comment:
It is not simple.

I would like to:
/?module=application_form&action=application_form

Please tell me the reason.